### DISCUSSION TITLE
Option to disable selection marker in PcmanFM-Qt

### DISCUSSION THREAD

**Discussion by maplee0** [lxqt/lxqt#2552] [Ideas]
Hello!
I'm not sure if similar thread was created in the past, sorry if it was.
It'd be nice to have ability to disable "selection marker" in the pcmanfm-qt settings.
Thanks!

**Comment by tsujan** [comment]
→ https://github.com/lxqt/pcmanfm-qt/issues/1187
As for adding options → https://github.com/lxqt/pcmanfm-qt/issues/1189#issuecomment-683436739
